ajax/libs/prunecluster/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "prunecluster",
+  "homepage": "https://sintef-9012.github.io/PruneCluster/",
+  "description": "Fast and realtime marker clustering for Leaflet",
+  "keywords": [
+    "map",
+    "clustering",
+    "leaflet",
+    "marker",
+    "realtime"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SINTEF-9012/PruneCluster.git"
+  },
+  "filename": "PruneCluster.js",
+  "npmName": "prunecluster",
```

Comment:
Let's use git auto-update, where we can see more build versions
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/SINTEF-9012/PruneCluster.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/!(*.d.ts)"
        ]
      }
    ]
  }
```

`**/!(*.d.ts)` is to exclude PruneCluster.d.ts which is not for browser use.